docs-ref-conceptual/azure-cli-configuration.md

```diff
@@ -88,6 +87,8 @@ When you provide a default value, that argument is no longer required by any com
 | __core__ | output | string | The default output format. Can be one of `json`, `jsonc`, `tsv`, or `table`. |
 | | disable\_confirm\_prompt | boolean | Turn confirmation prompts on/off. |
 | | collect\_telemetry | boolean | Allow Microsoft to collect anonymous data on the usage of the CLI. For privacy information, see the [Azure CLI Terms of Use](https://aka.ms/AzureCliLegal). |
+| | only\_show\_errors | boolean | Only show errors during command invocation. In other words, only errors will be written to `stderr`. It suppresses warnings from preview, deprecated and experimental commands. It is also available for individual commands with the `--only-show-errors` parameter. |
+| | no\_color | boolean | Disable color. Originally colored messages will be prefixed with `DEBUG`, `INFO`, `WARNING` and `ERROR`. This bypasses the issue of a third-party library where the terminal's color cannot revert back after a `stdout` redirection. |
```

Comment:
I am not sure whether we need to put the details of the context here, including the original issue. 

```suggestion
| | no\_color | boolean | Disable color. Originally colored messages will be prefixed with `DEBUG`, `INFO`, `WARNING` and `ERROR`. This bypasses the issue of the third-party library colorama ([#200](https://github.com/tartley/colorama/issues/200)) where the terminal's color cannot revert back after a `stdout` redirection. |
```